src/main/java/com/internet/shop/services/UserServiceImpl.java

```diff
@@ -4,6 +4,8 @@
 import com.internet.shop.lib.Inject;
 import com.internet.shop.lib.Service;
 import com.internet.shop.models.User;
+import com.internet.shop.util.HashUtil;
+
```

Comment:
empty line

Reply:
let's run `maven clean package` before push
also it would be nice to have travis configured

Reply:
fixed